public/editor/scripts/editor/js/fc/bramble-menus.js

```diff
@@ -88,11 +87,28 @@ define(function(require) {
       if(toggle) {
         $allowWSToggle.addClass("switch-enabled");
         bramble.enableWhiteSpace();
-        analytics.event("EnableWhiteSpaceIndicator");
+        analytics.event("EnableWhiteSpace");
       } else {
         $allowWSToggle.removeClass("switch-enabled");
         bramble.disableWhiteSpace();
-        analytics.event("DisableWhiteSpaceIndicator");
+        analytics.event("DisableWhiteSpace");
+    //set the AutoCloseTags toggle to reflect whether auto-close tags is enabled or disabled
```

Comment:
either indentation went very wrong here. and on the next several lines, or we lost the closing `}` that should go with line 94. Either way, something bad happened here and I'd be surprised if this file even parses correctly the way it is now.